recipes/sql-magic/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "sql-magic" %}
+{% set version = "0.0.4" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/sql_magic-{{ version }}.tar.gz
+  sha256: f54a08dda93b6b84f992ffc000d08ec491f44230a9f59e00aaf1b4d06c0f0324
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - findspark
+    - ipython
+    - jupyter
+    - pandas
+    - python >=3.6
+    - sqlparse
+    - traitlets
+
+test:
+  imports:
+    - sql_magic
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/pivotal-legacy/sql_magic
+  license_file: LICENSE.txt
+  license: Apache-2.0
+  license_family: Apache
+  summary: 'sql_magic is Jupyter magic for writing SQL to interact with Spark (or Hive) and relational databases. Query results are saved directly to a Pandas dataframe.'
```

Comment:
Can you make this shorter and have a longer `description` field

Reply:
Done!